### PR TITLE
Update tags for the website

### LINKED ISSUE #314: Recently merged icons need tags again!
Also it should be added in the design guide, that one should submit icons with some tags.

### PR DESCRIPTION
**Pull request by ericfennis**
Closes #314

### CODE REVIEW

File: tags.json [hunk 1]
```diff
@@ -234,6 +275,11 @@
     "paint",
     "color"
   ],
+  "bug": [
```
Comment (locness3) [inline tags.json:278]: `insect`?

File: tags.json [hunk 2]
```diff
@@ -348,6 +439,10 @@
     "source",
     "programming"
   ],
+  "code-2": [
```
Comment (locness3) [inline tags.json:442]: `html` and `xml`? as well for `code-1`

File: tags.json [hunk 3]
```diff
@@ -480,8 +575,12 @@
     "money",
     "payment"
   ],
-  "download": [],
-  "download-cloud": [],
+  "download": [
```
Comment (locness3) [inline tags.json:578]: *export?

File: tags.json [hunk 4]
```diff
@@ -540,56 +643,78 @@
...
+    "document"
   ],
   "file-plus-2": [
     "add",
     "create",
-    "new"
+    "new",
+    "document"
   ],
   "file-search": [
```
Comment (locness3) [inline tags.json:692]: `find` as well

**Review by locness3 — CHANGES_REQUESTED** [review]
Some suggestions, didnt go through the whole thing

File: tags.json (hunk 1 above)
Comment (ericfennis) [reply tags.json:278]: https://en.wikipedia.org/wiki/Insect

**Comment by ericfennis** [comment]
@locness3 updated!

File: tags.json (hunk 1 above)
Comment (locness3) [reply tags.json:278]: https://en.wikipedia.org/wiki/Bug?wprov=sfla1

> More broadly, any of many small, primarily terrestrial invertebrates, such as Insects

Comment (ericfennis) [reply tags.json:278]: yeah true, but it's just an tag to help people search for icons. 
So any related words or part of something or synonyms are welcome.

**Review by locness3 — APPROVED** [review]
lgtm.

**Pull request merged**